tests/test_client_functional.py

```diff
@@ -2453,6 +2460,126 @@ async def handler_redirect(request):
     assert data == body
 
 
+INVALID_URL_WITH_ERROR_MESSAGE = (
+    # yarl.URL.__new__ raises ValueError
+    ("http://:/", "http://:/"),
+    ("http://example.org:non_int_port/", "http://example.org:non_int_port/"),
+    # # yarl.URL.origin raises ValueError
+    ("http:/", "http:///"),
+    ("http:/example.com", "http:///example.com"),
+    ("http:///example.com", "http:///example.com"),
+    ("bluesky://d:i:d", "bluesky://d:i:d"),
+)
+
+NON_HTTP_URL_WITH_ERROR_MESSAGE = (
+    ("call:+3801234567", r"call:\+3801234567"),
```

Comment:
Let's use the realistic length here.
```suggestion
    ("call:+380123456789", r"call:\+380123456789"),
```